src/Symfony/Component/Uid/Tests/Fixture/CustomUlid.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Uid\Tests\Fixture;
```

Comment:
In most components, we named the dir "Fixtures".

Reply:
Thanks, renamed